commons/src/test/java/io/aiven/kafka/connect/common/source/task/FilenamePartitionTaskAssignmentTest.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright 2024 Aiven Oy
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.aiven.kafka.connect.common.source.task;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import org.apache.kafka.common.config.ConfigException;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+final class FilenamePartitionTaskAssignmentTest {
+
+    @Test
+    void partitionInFileNameDefaultAivenS3Sink() {
+        final TaskAssignment taskAssignment = new FilenamePartitionTaskAssignment();
+        taskAssignment.configureTask(1, 2, "logs-\\{\\{partition}}-\\{\\{start_offset}}");
+        assertThat(taskAssignment.isPartOfTask("logs-1-00112.gz")).isTrue();
+    }
+
+    @Test
+    void partitionLocationNotSetExpectException() {
+        final TaskAssignment taskAssignment = new FilenamePartitionTaskAssignment();
+        assertThrows(ConfigException.class,
```

Comment:
Can we avoid using JUnit assertions in favor of AssertJ? I know they are used in multiple places but would be good if we would avoid introducing more places. At some point will make a PR to get rid of them.

Reply:
Noted will make this update